packages/dds/tree/src/feature-libraries/chunked-forest/codec/nodeShape.ts

```diff
@@ -33,11 +33,11 @@ export class NodeShape extends Shape<EncodedChunkShape> implements NodeEncoder {
 	public constructor(
 		public readonly type: undefined | TreeNodeSchemaIdentifier,
 		public readonly value: EncodedValueShape,
-		public readonly fields: readonly KeyedFieldEncoder[],
+		public readonly fieldEncoders: readonly KeyedFieldEncoder[],
 		public readonly extraLocal: undefined | FieldEncoder,
```

Comment:
This name is actually out of date. We use to have local and global field keys, but that design got removed. While you are cleaning this up, fixing this to rename it to something like "encoderForOtherFields" would be nice.